packages/terra-application-utility/examples/index-examples/MockApplicationHeader.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import ApplicationHeaderLayout from 'terra-application-header-layout';
+import Popup from 'terra-popup';
+import Image from 'terra-image';
+import 'terra-base/lib/baseStyles';
+import LogoExample from 'terra-application-header-layout/examples/index-examples/LogoExample';
+import NavigationExample from 'terra-application-header-layout/examples/index-examples/NavigationExample';
+import ToggleExample from 'terra-application-header-layout/examples/index-examples/ToggleExample';
+import Avatar from './FallBackAvatar.svg';
```

Comment:
Since we have an Avatar component, can we rename the import from Avatar to FallbackAvatar? 

Also, Fallback is one word.